cli.go

```diff
@@ -246,13 +279,19 @@ func (cli *CLI) Run(args []string) int {
 			// shutdown
 			log.Printf("[INFO] (cli) signal received to initiate graceful shutdown: %v", sig)
 			cancel()
-			select {
-			case <-exitCh:
-				log.Printf("[INFO] (cli) graceful shutdown")
-				return ExitCodeOK
-			case <-time.After(10 * time.Second):
-				log.Printf("[INFO] (cli) graceful shutdown timed out, exiting")
-				return ExitCodeInterrupt
+			counter := 0
+			for {
+				select {
+				case <-exitCh:
+					counter++
+					log.Printf("[INFO] (cli) graceful shutdown api / controller")
```

Comment:
This is a bit confusing to see 2x since it's not clear on the order of which component shut down API or controller. I think it's more appropriate to log that the API has shutdown within this part of the code https://github.com/hashicorp/consul-terraform-sync/blob/7289c1e6ed0ad22f0b794258863b20b3eb50f7d9/api/api.go#L103-L106

That being said, I need to add a log that the controller shut down here 😛  https://github.com/hashicorp/consul-terraform-sync/blob/7289c1e6ed0ad22f0b794258863b20b3eb50f7d9/cli.go#L262
https://github.com/hashicorp/consul-terraform-sync/blob/7289c1e6ed0ad22f0b794258863b20b3eb50f7d9/cli.go#L245

Reply:
Yea, I can see how seeing this log twice would be confusing. I'll go ahead and add the logs you mentioned. I can also update this graceful shutdown log to be like "[INFO] (cli) graceful shutdown (1/2)" and then "[INFO] (cli) graceful shutdown (2/2)". Might still be weird to see though. What do you think?

Reply:
yeah a bit weird, could be perceived as shutdown attempts as opposed to different components shutting down.

Reply:
Good point about being misconstrued as shutdown attempts. I moved the graceful shutdown log so that it logs once (on the final exit) and log in the api. I ended up not adding the controller logs in cli since there are shutdown logs in the controller code themselves. It looked a little redundant when I tried it.

Reply:
Ohh right. Sorry about that!